### DISCUSSION TITLE
Add new command to allow players to pick their own chat color.

### DISCUSSION THREAD

**Discussion by NicoNekoDev** [Zrips/CMI#6686] [Feature Requests]
The idea is simple, add a new command (ex: `/cmi chatcolorpicker [color]`) to allow players to pick a color they want to chat with.
When doing with no arguments, it could show the same text as `/cmi colorpicker` command.
It could be used as placeholder in the messageFormat in the config.

**Comment by mrfloris** [comment]
Could we get "right-click" on the colors of `cmi colorpicker` to allow a user to set `cmi nick {#clicked}[playerName]` so we can display to them which colors they can use for nickname, and then they dont have to enter the values (and get it wrong) making the colorpicker effectively a nickname setter as well ??

**Reply by NicoNekoDev** [reply]
how about '/cmi nickcolorpicker [color]' about the same as my above request
